### PR TITLE
Test Cases added to token.rs

### PR DESCRIPTION
**Pull request by ankush1233**
added 3 more test cases for : is_start_tag(), is_any_start_tag(), is_text_token()

### CODE REVIEW

**Review by jaytaph — APPROVED** [review]

**Comment by jaytaph** [comment]
@ankush1233 Thanks for the PR.. Could you check the formatting of the code? This can be done easily by running `make fix` on the commandline. Also, not all the tests pass, because of a type in `assert`

File: src/html5/tokenizer/token.rs [hunk 1]
```diff
@@ -232,4 +232,36 @@ mod tests {
...
     }
+    #[test]
+    fn test_is_start_tag() {
+        let token = Token::StartTag{
+            name: "div".to_string(),
+            is_self_closing: false,
+            attributes: HashMap::new(),
+        };
+        assert!(token.is_start_tag("div"));
+        assret!(!token.is_start_tag("span"));
```
[the remark this answers is not in the document]
Comment (ankush1233) [reply src/html5/tokenizer/token.rs:243]: ohk i will rectify . thanks

**Comment by jaytaph** [comment]
@ankush1233 The assert type is fixed. If you could do a "make fix" so the formatting is correct, we can merge this pr

File: src/html5/tokenizer/token.rs [hunk 2]
```diff
@@ -232,4 +232,36 @@ mod tests {
...
+        let token = Token::StartTag{
+            name: "div".to_string(),
+            is_self_closing: false,
+            attributes: HashMap::new(),
+        };
+        assert!(token.is_start_tag("div"));
+        assert!(!token.is_start_tag("span"));
+    }
+    #[test]
+    fn test_is_any_start_tag() {
```
Comment (emwalker) [inline src/html5/tokenizer/token.rs:246]: Can we add a newline between each test and the one that follows?

```suggestion
    }

    #[test]
    fn test_is_any_start_tag() {
```

Comment (ankush1233) [reply src/html5/tokenizer/token.rs]: ohk i did that

**Pull request closed without merging**